.github/workflows/build.yaml

```
@@ -277,3 +338,635 @@ jobs:
+      recreate: false
+
+  ######################################
+  #             ISO                #
+  ######################################
+
+  online_iso_base:
+    name: Generate iso for base
+    needs: base
+    uses: ./.github/workflows/iso.yaml
+    secrets: inherit
+    permissions: *permissions
```

actionlint (1.7.8)

"permissions" section is alias node but mapping node is expected

(syntax-check)